### PR TITLE
Update demo

### PR DESCRIPTION
**Pull request by willread**
## Description

Update the demo page to reflect the latest functionality of the component.

## Motivation and Context

Work related to [this Trello card](https://trello.com/c/2U8KGpq1). A number of example cases have been added to the demo, including examples to demonstrate scaling and centering of videos, playlists with single / multiple videos, webm and mp4 videos and volume.

## How Has This Been Tested?

Demo page has been tested both locally and in Chrome / Electron players. It's been published to GCS [here](https://widgets.risevision.com/staging/pages/2019.07.30.08.15/src/rise-video.html) and is published in [this schedule](https://apps.risevision.com/schedules/details/affc198c-62ad-48df-8e72-bc9e13c3395e).

## Release Plan:

- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
  - Manual Test: Completed as noted above
  - Automated Test: Completed as noted above
  - Monitoring: No monitoring should be required
  - Rollback: The previous release has been tagged so rollback should be simple
  - Documentation: Usage documentation has been updated to reflect some minor changes
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
- No release checklist items were skipped

### CODE REVIEW

File: demo/src/rise-video.html [hunk 1]
```diff
@@ -29,21 +29,108 @@
...
+    }
+  </style>
 </head>
 
 <body>
+  <section class="video-container">
+    <h1>16:9</h1>
+
+    <div class="video-wrapper" style="width: 400px; height: 225px;">
+      <rise-video id="rise-video-01" files='["risemedialibrary-7fa5ee92-7deb-450b-a8d5-e5ed648c575f/rise-video-demo/Beach-Ball.mp4"]'>
```
Comment (santiagonoguez) [inline demo/src/rise-video.html]: you could also add a label to the rise-video components, so a nice label appears in settings.

**Comment by santiagonoguez** [comment]
@willread also, I think the demo should also include a non editable video, have you tested that flag with at least one player ?

File: demo/src/rise-video.html (hunk 1 above)
Comment (willread) [reply demo/src/rise-video.html]: @santiagonoguez I added a label to all components.

**Comment by willread** [comment]
@santiagonoguez Does it make more sense to add a non-editable component to the `example-video-component`, as this will actually be run inside apps?

**Pull request merged**